File: src/main/java/net/effize/bandlog/auth/application/exception/IllegalAuthenticationException.java
package net.effize.bandlog.auth.application.exception;

import net.effize.bandlog.common.exception.BandlogException;
import net.effize.bandlog.common.exception.ErrorCode;

public class IllegalAuthenticationException extends BandlogException {
    public IllegalAuthenticationException() {
        super(ErrorCode.of("ILLEGAL_AUTHENTICATION"), "인증 과정에 오류가 발생했습니다.", "Illegal authentication error", 500);
    }
}

File: src/main/kotlin/net/effize/bandlog/auth/domain/exception/IllegalAuthenticationException.kt
package net.effize.bandlog.auth.domain.exception

import net.effize.bandlog.common.exception.BandlogException
import net.effize.bandlog.common.exception.ErrorCode

class IllegalAuthenticationException : BandlogException(
    errorCode = ErrorCode.of("ILLEGAL_AUTHENTICATION"),
    userMessage = "인증 과정에 오류가 발생했습니다.",
    logMessage = "Illegal authentication error",
    httpStatusCode = 500
)
